run-integration-tests.bat

```diff
@@ -0,0 +1,9 @@
+:: we can't use makefile for windows because it depends on CI makefile which depends on shell
+
+:: compile sourced-ce
+if not exist build mkdir build
```

Comment:
is not `build/sourced-ce_windows_amd64` automatically created when runnuning `go build -o build/sourced-ce_windows_amd64/sourced.exe ./cmd/sourced` on windows? In Ubuntu it does.

Reply:
You're right, the dirs are created by `go build` itself. Addressed here: https://github.com/src-d/sourced-ce/pull/171/commits/e3b9e9e348fdf5aae52e629b8ed1a11c73053874.